VotingApplication/VotingApplication.Web/Api/Models/DBViewModels/PollRequestResponseModel.cs

```diff
@@ -7,6 +7,7 @@ namespace VotingApplication.Web.Api.Models.DBViewModels
     public class PollRequestResponseModel
     {
         public Guid UUID { get; set; }
+        public Guid ManageId { get; set; }
```

Comment:
I don't think we want to have the ManageId here. If i'm invited to a poll, I find out the UUID. If I then request from /api/poll/{UUID}, I can find out the ManageId and gain access to the management portal!